README.md

```diff
@@ -1,197 +1,37 @@
-# airpwr
+# Airpower
 
 A package manager and environment to provide consistent tooling for software teams.
 
-`pwr` provides declarative development environments to teams when traditional isolation and virtualization technologies cannot be employed. Calling `pwr shell` configures a consistent, local shell with the needed tools used to build and run software. This empowers teams to maintain consistentency in their build process and track configuration in version control systems (CaC).
+Airpower manages software packages using container technology and allows users to configure local PowerShell sessions to their need. Airpower seamlessly integrates common packages with a standardized project script for providing build commands which can be kept in source control for consistency across environments.
 
 # Requirements
 
-`pwr` requires the use of `$env:SystemDrive\Windows\System32\curl.exe` and `$env:SystemDrive\Windows\System32\tar.exe`, which are only available on Windows builds greater than `17063`.
-
-Use the following command in a `powershell` terminal to determine your build version.
-
-	[Environment]::OSVersion.Version
-
-PowerShell version 5.0+ is required. Windows 10 and 11 have version 5.1 installed by default.
-
-Use the following command in a `powershell` terminal to determine your PowerShell version.
-
-	$PSVersionTable.PSVersion
+Windows operating system with PowerShell version 5.1 or later.
 
 # Installing
 
-## PowerShell (recommended)
-
-Open a `powershell` terminal and execute the following command:
-
-	iex (& "$env:SYSTEMROOT\System32\curl.exe" -s --url 'https://raw.githubusercontent.com/airpwr/airpwr/main/src/install.ps1' | Out-String)
-
-The installer downloads the `pwr` cmdlet and puts its location on the user path.
-
-Run the `pwr` command afterwards to confirm that the installation was successful.
-
-	pwr help
-
-## Manually
-
-Save the `pwr.ps1` cmdlet to a file on your machine (`$env:AppData\pwr\cmd` is recommended), and add that location to the `Path` environment variable.
+To install Airpower, use the PowerShell command:
 
-# Configuration
-
-`pwr` can be configured entirely through its command line syntax; however, for convenience and to enable *Configuration as Code*, a file named `pwr.json` in the current or any parent directory is supported.
-
-`pwr.json` file's contents might look like:
-
-```json
-{
-  "packages": [
-    "jdk:8",
-    "python"
-  ],
-  "repositories": [
-    "airpower/shipyard",
-    "example.com/registry/v2/my/repo"
-  ],
-  "scripts": {
-    "name": "command"
-  }
-}
+```PowerShell
+Install-Module Airpower -Scope CurrentUser
```

Comment:
Not sure if you want to include this, but the following commands can be used to install Airpower without any user prompts (i.e. if you need to run it in a CI job):

```Powershell
Install-PackageProvider -Name NuGet -Force
Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
Install-Module Airpower -Scope CurrentUser
```

Reply:
I think it's worth putting in the README